backend/src/server.js

```diff
@@ -1,84 +1,75 @@
-require('dotenv').config({ path:  '.env' })
-const cors = require('cors');
-const express = require('express')
-const app = express()
+require('dotenv').config({ path: '.env' });
```

Comment:
does this inject into the process.env file, otherwise i don't see this use.

Reply:
That's what it does, yes. We're getting the SERVER_PORT value from the file on line 4